pairtaghighlighter/src/pair_tag_highlighter.c

```diff
@@ -319,6 +321,9 @@ static void findMatchingTag(ScintillaObject *sci, gint openingBracket, gint clos
     gboolean isTagOpening = is_tag_opening(sci, openingBracket);
     gchar *tagName = get_tag_name(sci, openingBracket, closingBracket, isTagOpening);
 
+    if (!tagName)
```

Comment:
In current implementation this check is redundant, because findMatchingTag() call is under the check and it is not invoked on empty brackets. 

If you prefer to have this check for safety, I would follow plugin's normal behavior: when pair is not found, original tag should be highlighted with NONMATCHING_PAIR_COLOR.